apps/allocations/app/components/App/App.js

```diff
@@ -2,21 +2,50 @@ import React, { useState } from 'react'
 import PropTypes from 'prop-types'
 
 import { useAragonApi } from '../../api-react'
-import { Button, Header, IconPlus, Main, SidePanel } from '@aragon/ui'
+import { Button, Header, IconPlus, Main, Modal, SidePanel } from '@aragon/ui'
 
 import { IdentityProvider } from '../../../../../shared/identity'
 import { Empty } from '../Card'
 import { NewAllocation, NewBudget } from '../Panel'
-import { Accounts, Payouts } from '.'
+import { AllocationsHistory, Budgets } from '.'
 
 const ASSETS_URL = './aragon-ui'
 
 const nameSorter = (a, b) => a.data.name.toUpperCase() > b.data.name.toUpperCase() ? 1 : -1
 
 const App = () => {
   const [ panel, setPanel ] = useState(null)
+  const [ modal, setModal ] = useState({ visible: false, budgetId: null })
   const { api, appState } = useAragonApi()
-  const { accounts = [], balances = [], payouts = [] } = appState
+  const {
+    // backend stub, remove
```

Comment:
We probably should not want to have this once it is presented for review unless the PR is work in progress

Reply:
It was meant for you to remove it after you get an idea of the data structures used